app/components/pullRequestPanel/pull.request.panel.component.ts

```diff
@@ -502,7 +502,8 @@ export class PullRequestPanelComponent {
       url: url,
       type: "GET",
       beforeSend: function (xhr) {
```

Comment:
Authorization headers now use access tokens instead of username and password hash